catalog/core/catalog-core-validationparser/src/test/groovy/ddf/catalog/validation/impl/ValidationParserSpecTest.groovy

```diff
@@ -691,4 +709,29 @@ class ValidationParserSpecTest extends Specification {
 }
 '''
 
+    String matchOneAttributeValidator = '''
+{
+    "validators": {
```

Comment:
is the value of the title the array of validators?  Maybe this comment should have been one line lower on 

> `+        "title": [`